src/containers/Home/dashboardHome.tsx

```diff
@@ -60,31 +61,7 @@ export const DashboardHome = () => {
                             {UIConstants.teamMemberTitle}
                         </Typography>
                     </AccordionSummary>
-                    <AccordionDetails>
-                        <CardBoxList />
-                    </AccordionDetails>
-                </Accordion>
-
-                <Accordion
-                    expanded={expanded === "panel4"}
-                    onChange={handleChange("panel4")}
-                >
-                    <AccordionSummary
-                        expandIcon={<ExpandMoreIcon />}
-                        aria-controls="panel4bh-content"
-                        id="panel4bh-header"
-                        sx={{
-                            backgroundColor: "lightgray",
-                            color: "black",
-                        }}
-                    >
-                        <Typography sx={{ width: "33%", flexShrink: 0 }}>
-                            {UIConstants.achievementsTitle}
-                        </Typography>
-                    </AccordionSummary>
-                    <AccordionDetails>
-                        <CardBoxList />
-                    </AccordionDetails>
+                    <AccordionDetails>Employee details</AccordionDetails>
```

Comment:
Pls add the string "Employee details" to constants file

Reply:
Changes are done as suggested